test/parallel/test-cluster-debugport-overflow.js

```diff
@@ -0,0 +1,18 @@
+var spawn = require('child_process').spawn;
+var cluster = require('cluster');
+var assert = require('assert');
+
+if (process.argv[2] == 'master') {
+   if (cluster.isMaster) {
+      cluster.fork().on('exit', function(code) {
+         process.exit(code);
+      })
+   } else {
+      process.exit(42);
+   }
+} else {
+   // iojs --debug-port=65535 test-cluster-debugport-overflow.js master
+   spawn(process.argv[0], ['--debug-port=65535', __filename, 'master']).on('close', function(code){
```

Comment:
Style nits: line > 80 columns and missing space before brace.  If you assign the arguments to a `var args` first, you stay under the limit.

Ditto for the line in lib/cluster.js; eyeballing it, it looks to be just over 80 columns.